qiskit/pulse/calibration_entries.py

```diff
@@ -316,11 +321,16 @@ def __init__(
     def _build_schedule(self):
         """Build pulse schedule from cmd-def sequence."""
         schedule = Schedule(name=self._name)
-        for qobj_inst in self._source:
-            for qiskit_inst in self._converter._get_sequences(qobj_inst):
-                schedule.insert(qobj_inst.t0, qiskit_inst, inplace=True)
-        self._definition = schedule
-        self._parse_argument()
+        try:
+            for qobj_inst in self._source:
+                for qiskit_inst in self._converter._get_sequences(qobj_inst):
+                    schedule.insert(qobj_inst.t0, qiskit_inst, inplace=True)
+            self._definition = schedule
+            self._parse_argument()
+        except QiskitError:
+            # When the play waveform data is missing in pulse_lib we cannot build schedule.
+            # Instead of raising an error, get_schedule should return None.
+            self._definition = IncompletePulseQobj
```

Comment:
Could we emit a warning (UserWarning?) here with the message from the `QiskitError`? Here we are treating a calibration loading error as equivalent to no calibration being provided and the user might want to know when there is a loading error.

Reply:
Fair enough. Added an error and test in [bf723eb](https://github.com/Qiskit/qiskit/pull/11397/commits/bf723eb8ca1b5f9aabd380f8a342f026aabd1478)